src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,23 @@
+import './Breadcrumbs.scss';
+
+interface IBreadCrumbsUrlItem {
+  href: string;
+  name: string;
+}
+
+interface IBreadCrumbsProps {
+  urls: IBreadCrumbsUrlItem[];
+}
+
+export const BreadCrumbs = (props: IBreadCrumbsProps) => {
+  const { urls } = props;
+  return (
+    <section>
+      <div className="line">
+        <h1>
+          {urls.map((item) => (item.href ? <a href={item.href}>{item.name}</a> : <span>{item.name}</span>))}
```

Comment:
you have to add "key" attribute (some details are here https://reactjs.org/docs/lists-and-keys.html)